cloud-formation/cloud-formation.yaml

```diff
@@ -169,18 +199,20 @@ Resources:
   EventbriteImagesELBDNSrecord:
     Type: 'AWS::Route53::RecordSet'
     Properties:
-      HostedZoneId: /hostedzone/Z1E4V12LQGXFEC
+      HostedZoneId: /hostedzone/Z8SGOO5042FR9
       Comment: CNAME for AWS ELB
-      Name: !Join
-        - .
-        - - !Ref SiteDomain
-          - origin.membership.guardianapis.com.
+      Name: !Sub ${App}-${Stage}.membership-event-management.gutools.co.uk.
       Type: CNAME
       TTL: '120'
       ResourceRecords:
         - !GetAtt
           - EventbriteImagesElasticLoadBalancer
           - DNSName
+  EventbriteImagesLogGroup:
+    Type: "AWS::Logs::LogGroup"
+    Properties:
+      LogGroupName: !Sub ${App}-${Stage}
```

Comment:
I don't think this is the log group name that [this script](https://github.com/guardian/amigo/blob/master/roles/cloud-watch-logs/files/configure-logs#L16) is expecting?

Reply:
Fixed